packages/kbn-search-api-panels/components/ingest_pipeline_panel.tsx

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useState, Fragment, useMemo } from 'react';
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+  EuiBadge,
+  EuiSuperSelect,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import type { Pipeline } from '@kbn/ingest-pipelines-plugin/public';
+
+interface OptionItem {
+  value: string;
+  inputDisplay: string;
+  dropdownDisplay: JSX.Element;
+}
+
+interface IngestPipelinePanelProps {
+  setSelectedPipeline: (pipeline: string) => void;
+  ingestPipelineData?: Pipeline[] | null;
+}
+
+const RECOMMENDED_INGEST_PIPELINE = 'search-default-ingestion';
```

Comment:
Can we reuse [DEFAULT_INGESTION_PIPELINE](https://github.com/elastic/kibana/blob/main/x-pack/plugins/serverless_search/public/application/constants.ts#L12) constant  instead?

Reply:
I moved the constant DEFAULT_INGESTION_PIPELINE to '@kbn/search-api-panels/constants' for broader usage. Let me know whether this makes sense.

Reply:
The default ingestion pipeline  is specific to `serverless_search`  project. So I think this variable doesn't have to be moved to shared package.

@TattdCodeMonkey to confirm if my understanding is correct.

Reply:
In that scenario, we should include the **serverless_search** plugin in the kbn_references of the **kbn-search-api-panels package**. This way, we can reuse the constant from there. There will be a circular dependency between **serverless_search** plugin and **kbn-search-api-panels package**.

Reply:
In this case, IMO You can pass  `DEFAULT_INGESTION_PIPELINE` as a prop to `IngestPipelinePanel` and parent components - `IngestData` and `ElasticsearchOverview` 

This way we won't have to move constant to '@kbn/search-api-panels/constants'

Reply:
@saarikabhasi Updated this part.